packages/slate-editor/src/components/ElementPlaceholder/Illustration.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import type { PropsWithChildren } from 'react';
+
+import styles from './ElementPlaceholder.module.scss';
+
+interface Props {}
+
+export function Illustration(props: PropsWithChildren<Props>) {
+    return <div className={styles.illustration}>{props.children}</div>;
+}
```

Comment:
As of my taste, these one-liner components do not deserve a separate file. We can just inline them into the `EmailPlaceholder.tsx` file. 

```tsx
function Illustration(props: { children: ReactNode }) {
    return <div className={styles.illustration}>{props.children}</div>;
}
```